### PR TITLE
Wardrobe chitchat refinements

### PR DESCRIPTION
**Pull request by Vreejack**
It's back to the original concept:

A rationalization of H's reaction to what players make her wear and her reasonable objection to going pantyless in some bottoms.

Johnny, how do I find you on Discord? Just installed it.

### CODE REVIEW

**Comment by asease1** [comment]
you can use this link for the discord
https://discord.gg/hCDa6U
will expire in a day

File: game/__Renpy__/wardrobe/wr_bottoms.rpy [hunk 1]
```diff
@@ -79,104 +106,212 @@ label equip_her_bottom:    #useful stuff: {w=0.9} {size=-2}text{/size} {b}text{/
...
-                if whoring >= 8:
-                    if whoring < 11:
-                        call her_main("Alright... sure, why not.","base","down")
-                        call her_main("Let me go and change real quick.","base","base")
-                    elif whoring < 20:
+                if whoring >= 8: #SUCCESS
+                    if whoring <= 10: #8-10
+                        call her_main("These are so cute.","grin","happy")
+                        #Due to whoring level, exposure check automatically fails for this bottom
+                        if not hermione_wear_panties:
```
Comment (asease1) [inline game/__Renpy__/wardrobe/wr_bottoms.rpy]: ads Test

**Comment by Vreejack** [comment]
Got it, thanks. Everyone use the same name?

**Comment by asease1** [comment]
yep pretty much, I am going to close this one for now

**Pull request closed without merging**